serve/package.go

```diff
@@ -31,16 +32,18 @@ This creates a directory with the plugin binaries, package.json and documentatio
 type PackageJSON struct {
 	SchemaVersion    int                `json:"schema_version"`
 	Name             string             `json:"name"`
+	Message          string             `json:"message"`
 	Version          string             `json:"version"`
 	Protocols        []int              `json:"protocols"`
 	SupportedTargets []TargetBuild      `json:"supported_targets"`
 	PackageType      plugin.PackageType `json:"package_type"`
 }
 
 type TargetBuild struct {
-	OS   string `json:"os"`
-	Arch string `json:"arch"`
-	Path string `json:"path"`
+	OS       string `json:"os"`
+	Arch     string `json:"arch"`
+	Path     string `json:"path"`
+	Checksum string `json:"checksum"`
```

Comment:
How are we going to use the `Checksum`? Shouldn't we also sign the checksum file

Reply:
@erezrokah they will be sent with HTTP PATCH after the upload.

Reply:
> @erezrokah they will be sent with HTTP PATCH after the upload.

Thanks @disq, but what do we do with them? Do we validate it?

Reply:
@erezrokah We pass them on to the client and after download the client should validate them (soon)

Reply:
> How are we going to use the Checksum?

We will store it in package.json here temporarily, and then read and upload it along with the binaries via the `cloudquery publish` command (which reads the package.json file). Then it'll be stored on the backend, and used to verify downloads of binaries from the CLI. 

> Shouldn't we also sign the checksum file

I think we can add this on the backend at a later stage. The only case I can think of is if an attacker were to do a MITM-type attack between us and the user downloading the plugin. To mitigate this I think the CLI will need to ship with our public key, to make sure the checksum file came from us.

Reply:
Yeah, mostly mentioning since we don't do any kind of checksum validation at the moment so maybe we can drop it altogether